src/scip_data.jl

```diff
@@ -26,50 +27,41 @@ mutable struct ManagedSCIP
     # Map from user-defined types (keys are <: AbstractSeparator) to the
     # corresponding SCIP objects.
     sepas::Dict{Any, Ptr{SCIP_SEPA}}
-
-    function ManagedSCIP()
-        scip = Ref{Ptr{SCIP_}}(C_NULL)
-        @SCIP_CALL SCIPcreate(scip)
-        @assert scip[] != C_NULL
-        @SCIP_CALL SCIPincludeDefaultPlugins(scip[])
-        @SCIP_CALL SCIP.SCIPcreateProbBasic(scip[], "")
-
-        mscip = new(scip, Dict(), Dict(), 0, 0, Dict(), Dict(), Dict())
-        finalizer(free_scip, mscip)
-    end
 end
 
-# Protect ManagedSCIP from GC for ccall with Ptr{SCIP_} argument.
-Base.unsafe_convert(::Type{Ptr{SCIP_}}, mscip::ManagedSCIP) = mscip.scip[]
+# TODO: all references to scipd / SCIPData need to be changed to (reference to optimmizer struct?) / SCIPData
```

Comment:
This is done, so the TODO comment may be dropped, right?

Reply:
Yes, the line has been deleted. Thanks for pointing it out.